kvbc/include/proof_processor/EmptyProofProcessor.h

```diff
@@ -0,0 +1,34 @@
+// Concord
+//
+// Copyright (c) 2023 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the sub-component's license, as noted in the
+// LICENSE file.
+
+#pragma once
+
+#include "IProofProcessor.h"
+#include "kvbc_app_filter/kvbc_key_types.h"
+
+namespace concord {
+namespace kvbc {
+
+class EmptyProofProcessor : public IProofProcessor {
```

Comment:
Is this a test class? If so, shouldn't it be in some test directory? If not, why is it needed?

Reply:
We want to have the possibility to not build the data structures related to verifiability. In deployments where they will not be required we don't want to pay the performance penalty of building and maintaining them. The current idea for the future is to instantiate the actual Proof processor based on run time configuration.